### ISSUE TITLE
register as Windows protocol handler

### ISSUE THREAD

**Issue by boringcactus** [LukeEmmet/GemiNaut#11]
i think https://docs.microsoft.com/en-us/windows/uwp/launch-resume/handle-uri-activation might be the relevant instructions? maybe? possibly?? idk C# is not something i'm super acquainted with

**Comment by LukeEmmet** [comment]
yes I am planning to implement this. Probably with user consent via a button in view->settings rather than to automatically register the protocol handler.

thanks for the link - looks handy

**Comment by jaynemo** [comment]
Having the option to register as the Gopher client handler could make sense too.
